spec/controllers/api/v0/orders_controller_spec.rb

```diff
@@ -161,19 +161,17 @@ module Api
                         as: :json
 
             expect(json_response['orders']
-              .map{ |o| o[:id] }).to match_array serialized_orders([order2, order3, order1, order4,
-                                                                    order7])
-                .map{ |o|
-                                                   o["id"]
-                                                 }
+              .pluck(:id)).to match_array serialized_orders([order2, order3, order1, order4,
+                                                             order7])
+                .pluck(:id)
```

Comment:
This is a bit confusing, and probably could be appended to the previous line.

Reply:
I would try to put the `expect()` on one line, and the `to ...` on the next line.
(also on next one below(

Reply:
It's been fixed here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/12599/commits/72640cfc2ac2d047e8dc657acc2b610b8b5e35ab

I've created a common memoized variable and it eliminated the need for multiline.